src/Mapping/Event/AdapterInterface.php

```diff
@@ -4,6 +4,8 @@
 
 use Doctrine\Common\EventArgs;
 use Doctrine\ORM\UnitOfWork;
+use Doctrine\Persistence\Event\LifecycleEventArgs;
```

Comment:
IMO, we should declare "doctrine/persistence" as requirement before using these classes.

Reply:
It's covered in https://github.com/doctrine-extensions/DoctrineExtensions/pull/2264 and https://github.com/doctrine-extensions/DoctrineExtensions/pull/2260, no need to add it on a third PR 😅